src/OpenLoco/Widget.cpp

```diff
@@ -8,6 +8,7 @@
 #include <cassert>
 
 using namespace OpenLoco::Interop;
+using OpenLoco::ImageIds::frame_background_image;
```

Comment:
This is a bit excessive for only one use of the item. Just use the qualified name in its one use.